lib/elastomer/client/scroller.rb

```diff
@@ -135,11 +135,11 @@ def add_sort_by_doc(query)
       query.merge(:sort => [:_doc])
     end
 
-  DEFAULT_OPTS = {
-      :index => nil,
-      :type => nil,
-      :scroll => "5m",
-      :size => 50,
+    DEFAULT_OPTS = {
+      index:   nil,
+      type:    nil,
+      scroll:  "5m",
```

Comment:
This might have something to do with our problem. 5 minutes is a pretty long time. Does the default need to be this big? It is `1m` in the docs.

OTOH, I don't think this changed so it's probably not necessary to change it...

Reply:
I had the very same though, and there is a commit where I changed it to `1m`. The philosophy of "change one thing at a time" came to mind and I reverted the default back to `5m`. But yes, it is rather long and we should investigate changing it.

Reply:
Yeah I agree we should lower it to `1m` as a follow up